index.html

```diff
@@ -13,10 +13,70 @@
       type="text/css"
     />
     <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>My Vlog</title>
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <div class="header">
+
+<div class="topnav">
```

Comment:
Try to indent your code so that it is more clear to read. Here, the div topnav is inside the div header. It would be easier to read if header was not indented and topnav was.